frontend/src/Map.js

```diff
@@ -0,0 +1,69 @@
+
+
+
+
+
+
+
+
+
+
+
+// import React, { useState, useEffect } from "react";
```

Comment:
I don't think we will need this as we won't be using this page. We will be importing multiple packages unnecessarily. So I think we can let this be.